src/Components/Components/src/Reflection/MemberAssignment.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.AspNetCore.Components.Reflection
     internal class MemberAssignment
     {
         public static IEnumerable<PropertyInfo> GetPropertiesIncludingInherited(
-            Type type, BindingFlags bindingFlags)
+            Type? type, BindingFlags bindingFlags)
```

Comment:
I wouldn't make this one nullable but have a nullable local variable declared instead, or just refactor to a do..while loop